airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/manifest.yaml

```diff
@@ -1,168 +1,1195 @@
+    request_headers:
+      Amazon-Advertising-API-ClientId: '{{ config["client_id"] }}'
+  offset_paginator:
+    type: "DefaultPaginator"
+    pagination_strategy:
+      type: OffsetIncrement
+      page_size: 100
+    page_size_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: count
+    page_token_option:
+      type: RequestOption
+      field_name: startIndex
+      inject_into: request_parameter
+  streams:
+    profiles:
+      type: DeclarativeStream
+      name: profiles
+      primary_key:
+        - profileId
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: v2/profiles?profileTypeFilter=seller,vendor
+          http_method: GET
+          use_cache: true
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
+          type: NoPagination
+
+    profiles_filtered:
+      type: DeclarativeStream
+      name: profiles
+      primary_key:
+        - profileId
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: v2/profiles?profileTypeFilter=seller,vendor
+          http_method: GET
+          use_cache: true
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+          record_filter:
+            condition: >-
+              {%- set requested_profiles = config.get("profiles", []) -%}
+              {%- set requested_marketplace_ids = config.get("marketplace_ids", []) -%}
+              {{ record["profileId"] in requested_profiles or record["accountInfo"]["marketplaceStringId"] in requested_marketplace_ids if requested_profiles or requested_marketplace_ids else False }}
+        paginator:
+          type: NoPagination
+    portfolios:
+      type: DeclarativeStream
+      name: portfolios
+      primary_key:
+        - portfolioId
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: v2/portfolios/extended
+          http_method: GET
+          request_headers:
+            Amazon-Advertising-API-Scope: "{{ stream_slice['profileId'] }}"
+            Amazon-Advertising-API-ClientId: '{{ config["client_id"] }}'
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: "profileId"
+              stream: "#/definitions/streams/profiles_filtered"
+              partition_field: "profileId"
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
+          type: NoPagination
+
+    sponsored_display_campaigns:
```

Comment:
Should Sponsored Display streams have an error handler too? It seems like the previous behavior was the default one so it might be fine here.